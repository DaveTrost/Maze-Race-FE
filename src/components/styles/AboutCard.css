.Devs {
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;
  
  h3 {
    color: white;
    font-size: 2.5em;
    font-family: "KARNIVOT";
    background: -webkit-linear-gradient(bottom, #ff00ee, #00c3ff);
    background: linear-gradient(to top, #ff00ee, #00c3ff); 
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
  }
  
  img {
    height: 100px;
    width: 100px;
  }

  p{
    word-break: normal;
  }

};