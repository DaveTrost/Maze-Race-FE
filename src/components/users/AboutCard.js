import React from 'react';
import PropTypes from 'prop-types';

const AboutCard = ({ name, image, desc }) => {

  return (
    <>
      <h3>{name}</h3>
      <img src={image}></img>
      <p>{desc}</p>
    </>
  );
};

AboutCard.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired,
  desc: PropTypes.string.isRequired 
};

export default AboutCard;
