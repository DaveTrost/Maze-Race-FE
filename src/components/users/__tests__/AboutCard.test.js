import React from 'react';
import { shallow } from 'enzyme';
import AboutCard from '../AboutCard.js';

describe('AboutCard component', () => {
  it('renders AboutCard', () => {
    const wrapper = shallow(<AboutCard />);
    expect(wrapper).toMatchSnapshot();
  });
});
