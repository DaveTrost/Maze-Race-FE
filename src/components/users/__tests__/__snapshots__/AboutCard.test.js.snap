// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutCard component renders AboutCard 1`] = `
<Fragment>
  <h3 />
  <img />
  <p />
</Fragment>
`;
